src/OrchardCore/Microsoft.AspNetCore.Mvc.Modules.Abstractions/ModelBinding/ModelStateDictionaryExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace Microsoft.AspNetCore.Mvc.Modules.ModelBinding
+{
+    public static class ModelStateDictionaryExtensions
+    {
+        public static void AddModelError(this ModelStateDictionary modelState, string prefix, string key, string errorMessage)
```

Comment:
You should copy paste the documentation from the official AddModelError, and add the new parameter to it.